### ISSUE TITLE
without actual paths, the info is not valuable

### ISSUE THREAD

**Issue by nutpantz** [IvanVolosyuk/diskusage#101]
![Screenshot_20240723-191436](https://github.com/user-attachments/assets/fe1f6d63-39ff-4a5d-83ca-53ddbb7ae7a2)

what is "system data", where is it, and what is the path..
I have root. so.....  
i need more information.
more information should be the default.

this look like like generatic  google thinks you don't need to know specifics. just trust google 
there is no stortage/media directory on my device and no storage/apps
and nothing even close to "system data"

android 14 P8P

**Comment by sconim** [comment]
I agree. The app should show the actual folders, not an (unknown) reevaluation of allocated space. 

See X-Plore Disk Map function as a model.
